shopinvader_cart_reminder/models/sale_order.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from datetime import timedelta
+
+from odoo import api, fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    reminder_sent_datetime = fields.Datetime(
+        help="Date/hour of last  (automatic) reminder", default=False
```

Comment:
`"Date/hour of last  (automatic) reminder" ->  "Pending cart reminder sent date/hour"`